src/guides/v2.3/get-started/web-api-functional-testing.md

```diff
@@ -106,3 +106,9 @@ interface AdapterInterface
 1. Configure your DB connection and install settings in `/dev/tests/api-functional/config/install-config-mysql.php`. Specify the Magento database. The base URL to access this Magento instance must be the same specified in the `phpunit_rest.xml` or `phpunit_soap.xml` file.
 
 1. Run `phpunit` using the `/dev/tests/api-functional/phpunit_rest.xml` or `/dev/tests/api-functional/phpunit_soap.xml` configuration file.
+
+Example: 
```

Comment:
Hello @nuzil.
Could you please remove the trailing space in order to fix the failed test?
Thank you!
```suggestion
Example:
```

Reply:
I was faster then you :-D

Reply:
Great, thank you :)